### DISCUSSION TITLE
Can't find logs on non-docker installation

### DISCUSSION THREAD

**Discussion by jhgarrison** [requarks/wiki#6248] [Help / Questions]
I installed wiki.js 2.5.294 in Debian 11, not in docker.  According to what I find in the docs, the log should be in /var/wiki/logs but there's no such directory.  Where are the logs in a non-docker installation?

I'm getting 500 errors but the error messages in the nginx logs are not enough to pinpoint the issue.

**Comment by jhgarrison** [comment]
Anybody?

**Comment by NGPixel** [comment]
The logs are from the process output.

So if you run `node server`, it will be output to the console. If you're using a service file, you need to configure it to save the process output to the file of your choice.

**Reply by pathak-ashish** [reply]
Apologies if it sounds dumb, but I'm facing the same issue and I really cannot see the logs on the console / terminal.
Running it via `node server`.
The logs I'm talking about are the query executions which shows the PostgreSQL queries being run, `console.log()` outputs written in resolver logic, etc.
How do I enable those? I even have `logLevel: verbose` in config.yaml, not sure if that would help.
Thanks in advance for any light you can shed on this.

**Reply by NGPixel** [reply]
You need to set the environment variable `NODE_ENV` to `development` in order to see the SQL queries in the logs.
